packages/cli/commands/project/upload.js

```diff
@@ -50,6 +58,25 @@ exports.handler = async options => {
     message
   );
 
+  if (result.error) {
+    logApiErrorInstance(
+      result.error,
+      new ApiErrorContext({
+        accountId,
+        projectName: projectConfig.name,
```

Comment:
Is there a reason we're logging the error here but not in `dev`?

Reply:
I think it was for backwards compatibility with how it worked before. But I think I can actually remove this 👍